web/locales/de.json

```diff
@@ -91,7 +91,8 @@
         "temporaryDataOutage": "Live-Daten sind momentan leider nicht verfügbar",
         "production": "Produktion",
         "consumption": "Verbrauch",
-        "cpinfo": "<b>Verbrauch</b> bezieht Importe und Exporte mit ein, <b>Produktion</b> ignoriert sie"
+        "cpinfo": "<b>Verbrauch</b> bezieht Importe und Exporte mit ein, <b>Produktion</b> ignoriert sie",
+        "selectLanguage": "Sprache auswählen"
```

Comment:
"Sprache auswählen" is difficult to find in the page for people that don't speak German (there is more chance that if you want to switch language: you are not fluent in the current language). 
Maybe we can use a symbol (could be the country flag but it's not the best, language are not link to one country and it could be difficult if you don't know the flag) or simply use the English word

Reply:
I wanted the languages in their own languages, but I didn't know how to translate all those names.

Reply:
If you look in the screenshot I send you will see that there is a translate symbol. The text you mentioned is only the tooltip.